easybuild/easyconfigs/f/file/file-5.41-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'ConfigureMake'
+
+name = 'file'
+version = '5.41'
```

Comment:
Looks like 5.43 is out so lets update it

Reply:
Added easyconfig file for file-5.43-GCCcore-11.3.0.eb; did you want it submitted as a separate pr?

Reply:
well , actually i wanted *only* that version. I see no reason for the 5.41 GCCcore-11.3.0 version to exist?